SAPHana/bin/SAPHanaSR-manageAttr

```diff
@@ -670,6 +678,24 @@ chk_multi_target_requirements() {
     return $chk
 }
 
+# check, if the cluster or the msl and cln resource is in maintenance
+chk_for_maintenance() {
+    # check maintenance mode of cluster
+    # msl resource is reported unmanaged, but cln resource not
+    if crm_mon -r1 | grep "Resource management is DISABLED" >/dev/null 2>&1; then
+        cluInMaint="true"
+    fi
+    # check maintenance mode of msl and cln resource
+    resC="msl_.*_${RSID}_HDB"
+    resT="cln_.*_${RSID}_HDB"
+    if crm_mon -r1 | grep "$resC" | grep unmanaged >/dev/null 2>&1; then
```

Comment:
grep could be dangerous, if $resC is a substring of an other ressource.
We also could not say, that the resource name needs to follow our name convention.
The resource must be found via the type (ocf:suse:SAPHanaController), the SID and (maybe the ino)
Same for resT.

Reply:
I have reworked the function. I still use grep, but with the full mls_ or cln_ name retrieved from the cluster configuration

Reply:
as the current pushed fix only use the 'sid' to get the resource names, I have a working fix, which uses the combination of 'sid' and 'ino' to get the matching resource name.
I can provide this fix, but that means, that the option '--ino' will be mandatory for the script execution.